com.unity.render-pipelines.high-definition/Runtime/Material/Unlit/Unlit.shader

```diff
@@ -94,6 +94,11 @@ Shader "HDRP/Unlit"
         // Debug constants must be exposed as properties so the shader is compatible
         // with the SRP batcher
         [HideInInspector] _UnlitColorMap_MipInfo("_UnlitColorMap_MipInfo", Vector) = (0, 0, 0, 0)
+
+        
+        [HideInInspector][NoScaleOffset]unity_Lightmaps("unity_Lightmaps", 2DArray) = "" {}
```

Comment:
I know you already answer to this question in the past, but I don't remind why this can't be declare in shadervariable.hlsl like we do for regular lightmaps. Guess it was something related to DOTS architoecture but would like to confirm.

Reply:
ah sorry, just see it was the case, so what is the purpose of those declaration here?

Reply:
if it is required you miss layeredLit, layeredlit tessellation and axf.shader

Reply:
Also note: Unlit don't have lightmaps.

Reply:
these are in the shadervariables.hlsl but need to also be in the property block otherwise they can't be bound to the SRP batcher. It's more exposed in userland and not relying on old c++ side things so some rules are different.